src/test/java/com/getindata/connectors/http/internal/table/lookup/JavaNetHttpPollingClientConnectionTest.java

```diff
@@ -127,11 +131,13 @@ void shouldQuery200WithParams() {
         JavaNetHttpPollingClient pollingClient = setUpPollingClient(getBaseUrl());
 
         // WHEN
-        RowData result = pollingClient.pull(lookupRowData).orElseThrow();
+        Collection<RowData> results = pollingClient.pull(lookupRowData);
```

Comment:
Does this mean that the existing behaviour for a non-array reult would change to now me an array with one element? If so this would have a migration impact.